inc/core/styles/frontend.php

```diff
@@ -51,6 +51,7 @@ public function __construct() {
 		$this->setup_blog_colors();
 		$this->setup_form_fields_style();
 		$this->setup_single_post_style();
+		$this->setup_single_page_style();
```

Comment:
Can we have a single function here that uses the same subscribers and just changes meta based on context?